python/tvm/tir/schedule/testing.py

```diff
@@ -16,14 +16,33 @@
 # under the License.
 # pylint: disable=dangerous-default-value
 """Testing utilities for the TensorIR schedule API"""
-from typing import Sequence, Union
+from typing import Any, Sequence, Union
 
 import tvm
 from tvm.ir import IRModule, assert_structural_equal
 from tvm.tir import PrimFunc
 from tvm.tir.schedule import Schedule, Trace
 
 
+def assert_structural_equal_gs(
```

Comment:
shall we expand `gs` to `global_symbol`? feel free to push back if it's a huge ask (like affecting 80 files)

Reply:
I'm not a fan of the abbreviation, but it will probably break a lot of formatting haha. I'll see what will happen

Reply:
It was actually a very easy change via VSCode's "rename" shortcut and applying `black` to all files affected (only 29). :+1: